doc/users/whats_new/deprecations.rst

```diff
@@ -0,0 +1,5 @@
+Deprecations
+------------
+
+- The "Vega" colormaps are deprecated in Matplotlib 2.0.1 and will be removed
```

Comment:
Might be worth mentioning that they are simply being renamed, and that the "tab" colormaps are exactly the same as the now-old "vega" colormaps.